eds/interfaces/task.py

```diff
@@ -13,3 +13,8 @@ class Task(Plugin):
         "type": "object",
         "properties": {}
     }
+
+
+PluginInterface.register(Task)
+assert issubclass(Task, PluginInterface)
+assert isinstance(Task(), PluginInterface)
```

Comment:
these 3 lines don't make sense to me in the interface classes.   The `Task` interface extends the `Plugin` interface.  it's not an implementation.   I think these lines should be removed from all the interface classes.